firmware/controllers/map_averaging.cpp

```diff
@@ -170,6 +178,18 @@ static void endAveraging(void *arg) {
 #if EFI_PROD_CODE || defined(__DOXYGEN__)
 	v_averagedMapValue = adcToVoltsDivided(
 			mapAccumulator / mapMeasurementsCounter);
+	if (mapMinBufferLength > 1) {
```

Comment:
yes, you are right about one less % and one less for loop with zero invocation count

but in my opinion the benefits is in complexity reduction, I am sure that this code is not the bottleneck, we definitely have MUCH bigger ones somewhere else. So yes, let's make is simpler and more reliable as a consequence.